stanford-earth/matson
# EARTH-1410 Removing color override for Editorial Sidebar image caption.

# READY FOR REVIEW

# Summary
- Remove override that was making the image caption for Editorial Sidebar to low contrast

# Needed By (Date)
- Next push

# Steps to Test

1. Navigate to /news/climate-change-has-worsened-global-economic-inequality
2. Scroll down to the "Lifted up or dragged down" Editorial Sidebar
3. Confirm that the caption text has sufficient contrast

# Associated Issues and/or People
- JIRA ticket SE3: EARTH-1410

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
